src/lints/enum_struct_variant_field_missing.ron

```diff
@@ -71,4 +71,10 @@ SemverQuery(
     },
     error_message: "A publicly-visible enum has a struct variant whose field is no longer available under its prior name. It may have been renamed or removed entirely.",
     per_result_error_template: Some("field {{field_name}} of variant {{enum_name}}::{{variant_name}}, previously in file {{span_filename}}:{{span_begin_line}}"),
+    witness: (
+        hint_template: r#"match value {
+    {{ join "::" path }}::{{ variant_name }}{ {{ field_name }}, .. } => (),
```

Comment:
nit: we want to output code formatted the same way as `cargo fmt` would have formatted it, so that means a space before the destructuring
```suggestion
    {{ join "::" path }}::{{ variant_name }} { {{ field_name }}, .. } => (),
```